nextflow.config

```
@@ -405,70 +441,111 @@ set -C # No clobber - prevent output redirection from overwriting files.
 // Disable process selector warnings by default. Use debug profile to enable warnings.
 nextflow.enable.configProcessNamesValidation = false
 
-def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_timeline_${params.trace_report_suffix}.html"
 }
 report {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_report_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_report_${params.trace_report_suffix}.html"
 }
 trace {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${params.trace_report_suffix}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
 
 manifest {
     name            = 'nf-core/funcscan'
     author          = """Jasmin Frangenberg, Anan Ibrahim, Louisa Perelo, Moritz E. Beber, James A. Fellows Yates"""
+    // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        [
+            name: 'Jasmin Frangenberg',
+            affiliation: 'Leibniz Institute for Natural Product Research and Infection Biology Hans Knöll Institute, Jena, Germany',
+            email: 'jasmin.frangenberg@leibniz-hki.de',
+            github: 'jasmezz',
+            contribution: [author, maintainer],
```

```suggestion
            contribution: ['author', 'maintainer'],
```

I think the categories need to be in quotes